contracts/router/src/contract/query.rs

```diff
@@ -35,6 +35,10 @@ pub fn chains(
         .collect()
 }
 
+pub fn is_enabled(deps: Deps) -> bool {
+    STATE.load(deps.storage).unwrap_or(State::Disabled) == State::Enabled
```

Comment:
this is not necessarily that it's not set, but that there is an issue with the storage access. So since I have to deal with a possible error anyway I thought it made sense to disable the contract if there are any issues